spec/jobs/education_form/create_daily_spool_files_spec.rb

```diff
@@ -108,6 +108,7 @@ def perform_with_frozen_time
 
     it 'writes files out over sftp' do
       ENV['EDU_SFTP_HOST'] = 'localhost'
```

Comment:
Can we use ClimateControl for this? @kreek introduced it in some of the MVI specs